wgpu-types/src/lib.rs

```diff
@@ -3,7 +3,7 @@
  */
 
 #![allow(
-    // The intra doc links to the wgpu crate in this crate actually succesfully link to the types in the wgpu crate, when built from the wgpu crate.
+    // The intra doc links to the wgpu crate in this crate actually successfully link to the types in the wgpu crate, when built from the wgpu crate.
```

Comment:
Thank you for this fix, but this is actually extremely wrong now... Will fix independently.